### PR TITLE
Update install/uninstall info for ARM

### LINKED ISSUE #27135: Indicate new install locations with .NET 6 and Arm64 on Mac
I don't know the details but I'm seeing questions around this and I don't think we're covering this change on the docs.

For example: https://github.com/dotnet/core/issues/6930

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7f2842ce-ee53-ec35-90c6-791a952dfc3b
* Version Independent ID: 79ec96d5-9c2c-1eaa-feb8-40b0afee6a30
* Content: [Check installed .NET versions on Windows, Linux, and macOS - .NET](https://docs.microsoft.com/en-us/dotnet/core/install/how-to-detect-installed-versions)
* Content Source: [docs/core/install/how-to-detect-installed-versions.md](https://github.com/dotnet/docs/blob/main/docs/core/install/how-to-detect-installed-versions.md)
* Product: **dotnet**
* Technology: **dotnet-install**
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

### PR DESCRIPTION
**Pull request by adegeo**
## Summary

- Added sections that describe how ARM64 installs differ from x64 installs on Mac/Windows.
- Added compatibility table for Mac/Windows related to runtimes/SDKs supported with ARM64.
- Added a little more info in the uninstall article.

Fixes #27135

[Internal preview: mac](https://review.docs.microsoft.com/en-us/dotnet/core/install/macos?branch=pr-en-us-27186)
[Internal preview: windows](https://review.docs.microsoft.com/en-us/dotnet/core/install/windows?branch=pr-en-us-27186&tabs=net60)
[Internal preview: uninstall](https://review.docs.microsoft.com/en-us/dotnet/core/install/remove-runtime-sdk-versions?branch=pr-en-us-27186&pivots=os-windows)

@mairaw @svick

### CODE REVIEW

File: docs/core/install/windows.md [hunk 1]
```diff
@@ -82,6 +82,39 @@ The SDK is used to build and publish .NET apps and libraries. Installing the SDK
...
+
+### What's supported
+
+The following table describes which versions of .NET are supported on an ARM-based Windows PC:
+
+| .NET Version | Architecture | SDK | Runtime | [Path conflict](#path-conflicts) |
+|--------------|--------------|-----|---------|----------------------------------|
+| 6.0          | ARM64        | Yes | Yes     | No                               |
+| 6.0          | x64          | Yes | Yes     | No                               |
+| 5.0          | ARM64        | No  | Yes     | N/A                              |
```
Comment (mairaw) [inline docs/core/install/windows.md:99]: I do see Arm64 SDK for .NET 5 at https://dotnet.microsoft.com/download/dotnet/5.0

File: docs/core/install/windows.md [hunk 2]
```diff
@@ -82,6 +82,39 @@ The SDK is used to build and publish .NET apps and libraries. Installing the SDK
 > [!div class="button"]
 > [Download .NET SDK](https://dotnet.microsoft.com/download/dotnet)
 
+## ARM-based Windows PCs
```
Comment (mairaw) [inline docs/core/install/windows.md:85]: I switched to use Arm on the website with @richlander's advice.
https://www.arm.com/

File: docs/core/install/windows.md (hunk 1 above)
Comment (adegeo) [reply docs/core/install/windows.md:99]: The download page seems to contradict what was documented on the desig doc:
https://github.com/dotnet/designs/blob/main/accepted/2021/x64-emulation-on-arm64/x64-emulation.md#update

Comment (mairaw) [reply docs/core/install/windows.md]: @richlander do we need to update the design doc in this case?

**Comment by adegeo** [comment]
@IEvangelist @mairaw this is ready for review. It would be better to get this published even though Rich is on vacation.

File: docs/core/install/macos.md [hunk 3]
```diff
@@ -127,17 +127,50 @@ export PATH=$PATH:$DOTNET_ROOT
 
 This approach lets you install different versions into separate locations and choose explicitly which one to use by which application.
 
+## Arm-based Macs
```
Comment (IEvangelist) [inline docs/core/install/macos.md:130]: Is Macs correct? Or should it be macOS

**Review by IEvangelist — APPROVED** [review]
Looks good to me, :shipit: when you're ready.

File: docs/core/install/macos.md (hunk 3 above)
Comment (adegeo) [reply docs/core/install/macos.md:130]: @IEvangelist  I wasn't sure, but it seems to be how its talked about:

https://appleinsider.com/inside/arm-mac

**Pull request merged**